test/boost/cdc_test.cc

```diff
@@ -460,18 +465,28 @@ SEASTAR_THREAD_TEST_CASE(test_pre_image_logging) {
 
                 auto pre_image = to_bytes_filtered(*rows, cdc::operation::pre_image);
                 auto second = to_bytes_filtered(*rows, cdc::operation::update);
+                auto post_image = to_bytes_filtered(*rows, cdc::operation::post_image);
 
                 if (!enabled) {
                     BOOST_REQUIRE(pre_image.empty());
+                    BOOST_REQUIRE(post_image.empty());
                 } else {
                     sort_by_time(*rows, second);
+                    sort_by_time(*rows, pre_image);
```

Comment:
This change fixes a bug in the pre_image test?

Reply:
It pre-empts a potential bug.